src/store/actions/test.js

```diff
@@ -2,6 +2,9 @@
 
 import * as actions from './';
 import ActionTypes from './ActionTypes';
+// Gutenberg imports
+import { getBlockType, serialize, createBlock } from '@wordpress/blocks';
```

Comment:
`getBlockType` and `serialize` don't seem to be used anymore, can we remove them?

Reply:
nice catch! addressed in 54220f8

Reply:
Thanks for the change @mzorz ! Apparently, the eslint configuration is actually faulty at the moment since this one should be caught during `yarn lint` (which is already part of the Travis tests).

I will produce a separate PR with the eslint fix (and also fix there a few other violations) so to not pollute this PR, cool?

Reply:
Thanks for explaining and describing where things are / need to be and follow up actions🙇 thank you! Also think the decision to make a separate PR for eslint fixes is in the right direction 👍